hax/setup.py

```diff
@@ -163,7 +163,8 @@ def is_runtime_comment(line: str) -> bool:
     setup_requires=reqs['build'],
     install_requires=reqs['runtime'],
     entry_points={
-        'console_scripts': ['hax=hax.hax:main', 'q=hax.queue.cli:main']
+        'console_scripts': ['hax=hax.hax:main', 'q=hax.queue.cli:main',
```

Comment:
I think that we need to choose either of options:
1. Generate the executable via setup.py (+ modify `hare.spec` - see `%install` section and `hare_setup`)
2. Create wrapper script

If we have `utils/generate_coverage`, do we need to generate this `hare_cov`?

cc @mssawant

Reply:
@knekrasov, 
Today I have removed .py3venv and again created hare_cov binary. Executing it without activate the environment and  found it is showing error ( Not sure how it is possible, Friday it was working and yes I have set the pythonpath and path env too).
![image](https://user-images.githubusercontent.com/86037325/129513333-6279b600-6451-4097-b60b-94956b2ddb64.png)

=> Again hctl script also we are activating the virtual environment. So its looks like wrapper script is the best option here.

![image](https://user-images.githubusercontent.com/86037325/129513240-9b743005-366f-4a76-a4e6-4a57b9af6494.png)

=> With wrapper script we can have command lines for xml and html coverage type too.

Reply:
@knekrasov, we can not modify the hare.spec, right? As per our discussion, I understood we can not modify the hare.spec.

Reply:
> we can not modify the hare.spec, right? As per our discussion, I understood we can not modify the hare.spec.

@ajaykumarptl Of course we can modify hare.spec. Moreover, we should if we want to use autogenerated scripts.

> => In hctl script also, we are activating the virtual environment. So its looks like wrapper script is the best option here if we need to enable virtual environment.

I'm ok with this way. My point is that if you implement a wrapper script anyway, then you don't need auto generated scripts. Instead, you can invoke your python file right from your wrapper script (`source <virtualenv>/bin/activate && python ./coverage.py`).

Reply:
@knekrasov / @mssawant,

As executable generation won't work in dev environment. Removing executable generation code and going ahead with the wrapper script.

Reply:
@knekrasov / @mssawant ,

As we are using wrapper script, moving hare_cov package to the utils.